### PR TITLE
Cat

### PR DESCRIPTION
**Pull request by seanxwzhang**
I've added a reference for the command "cat"
will add more stuff if needed

### CODE REVIEW

File: _commands/basics/cat.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+cat
+--
+
+`cat` is used to concatenate files and print on the standard output
+
+~~~ bash
+$ echo haha > file1
+$ echo lol > file2
+$ cat file1 file2
```
Comment (mterwill) [inline _commands/basics/cat.md]: Show the output here or this example doesn't tell much at a glance..

File: _commands/basics/cat.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+#### `cat -n`
+
+number all output lines
+
+~~~ bash
+$ echo haha > file1
+$ echo lol > file2
+$ cat file1 file2
+$ 1  haha
```
Comment (mterwill) [inline _commands/basics/cat.md]: ``$`` indicates a prompt, rather than output

**Comment by mterwill** [comment]
Actually, this was claimed in #15. @jattisha, have you started working on ``cat``? If not, maybe you could let @seanxwzhang take it.

**Comment by seanxwzhang** [comment]
I've modified the commit as requested, let me know if I can take the credits for this reference

**Comment by mterwill** [comment]
@jattisha, please confirm.

**Comment by jattisha** [comment]
Sorry I didn't see the first message, I thought it was an autograder email. I've been working on it but haven't had time to submit, so it would be really irritating if I had to redo it and choose a different one. Should have the complete one up by tomorrow.

Jovan H. Attisha
University of Michigan 2017 | College of Engineering 
Bachelor of Science in Computer Engineering
Treasurer, Chaldean American Student Association

**Comment by mterwill** [comment]
Claimed in #15, implemented in #84

**Pull request closed without merging**